trunner/test_runner.py

```diff
@@ -143,11 +143,12 @@ def parse_tests(self) -> Sequence[TestOptions]:
     def flash(self) -> TestResult:
         """Flashes the device under test."""
 
-        print("Flashing an image to device...")
-
         # report flashing as a test result to include in export (especially useful when failed)
         result = TestResult("flash")
 
+        self._print_test_header_begin(result)
+        print("Flashing an image to device...")
```

Comment:
```suggestion
```

It may be a bit less readable, but for consistency it would be better to have `flash: OK`, now it looks like this:

![Screenshot from 2024-04-16 11-52-33](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/77304431/2badd1bf-1526-405f-835a-1eb65f834c96)